src/Components/MainPage/PositionDetailed/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import useQuery from 'Hooks/useQuery';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { getPositionById } from 'redux/positions/thunks';
+import { cleanSelectedItem } from 'redux/interviews/actions';
+import { useHistory } from 'react-router-dom';
+
+function PositionDetailed() {
+  const history = useHistory();
+  const query = useQuery();
+  const dispatch = useDispatch();
+  const selectedItem = useSelector((store) => store.positions.selectedItem);
+
+  const positionId = query.get('_id');
+  useEffect(() => {
+    if (positionId) {
+      dispatch(getPositionById(positionId));
+    }
+  }, []);
+
+  useEffect(() => {
+    return () => {
+      dispatch(cleanSelectedItem());
+    };
+  }, []);
+
+  return (
```

Comment:
Every field should have a title: "Position", "Client", "Opened at"

Reply:
Check Computrabajo or any other website. Its quite rare to have a title. Perhaps a description should be added